src/nuclio/functions/version/version-configuration/tabs/version-configuration-resources/version-configuration-resources.component.js

```diff
@@ -90,7 +90,24 @@
             ],
             value: ValidationService.getValidationRules('k8s.qualifiedName')
         };
+        ctrl.podTolerationsOptions = [
+            {
+                id: 'allow',
+                name: 'Allow',
+                tooltip: $i18next.t('functions:TOOLTIP.POD_TOLERATIONS.ALLOW', {lng: lng})
+            },
+            {
+                id: 'constrain',
+                name: 'Constrain',
+                tooltip: $i18next.t('functions:TOOLTIP.POD_TOLERATIONS.CONSTRAIN', {lng: lng})
+            },
+            {
+                id: 'none',
+                name: 'none"/"'
```

Comment:
`None` seems like the better choice
```suggestion
                name: 'None'
```